src/ameba/rule/metrics/cyclomatic_complexity.cr

```diff
@@ -0,0 +1,33 @@
+module Ameba::Rule::Metrics
+  # A rule that disallows methods with a cyclomatic complexity higher than `MaxComplexity`
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Metrics/CyclomaticComplexity:
+  #   Enabled: true
+  #   MaxComplexity: 10
+  # ```
+  #
+  struct CyclomaticComplexity < Base
+    properties do
+      enabled false
```

Comment:
Let's enable that by default. Just remove this line, please.

Reply:
Sure, happy to ✌️